docs/reference/settings.yaml

```diff
@@ -764,11 +764,11 @@ settings:
           - Optional
         doc: |
           Setting `use_proxy_protocol` will configure Pomerium to require the [HAProxy proxy protocol](https://www.haproxy.org/download/1.9/doc/proxy-protocol.txt) on incoming connections. Versions 1 and 2 of the protocol are supported.
-      - name: "Envoy Admin Options"
-        keys: ["envoy_admin_options"]
+      - name: "Envoy Bootstrap Options"
+        keys: ["envoy_admin_address","envoy_admin_access_log_path","envoy_admin_profile_path","envoy_bind_config_freebind","envoy_bind_config_source_address"]
         attributes: |
           - Environment Variable: `ENVOY_ADMIN_ADDRESS`, `ENVOY_ADMIN_ACCESS_LOG_PATH`, `ENVOY_ADMIN_PROFILE_PATH`
-          - Config File Keys: `envoy_admin_address`, `envoy_admin_access_log_path`, `envoy_admin_profile_path`
+          - Config File Keys: `envoy_admin_address`, `envoy_admin_access_log_path`, `envoy_admin_profile_path`, `envoy_bind_config_freebind`, `envoy_bind_config_source_address`
           - Type: `string`
           - Optional
         doc: |
```

Comment:
The `doc` section should probably work in a link to the [cluster manager proto](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/bootstrap/v3/bootstrap.proto.html#config-bootstrap-v3-clustermanager) for these options.

/cc @alexfornuto

Reply:
@travisgroth: Updated, should be reviewed by an SME.